code/network/multi_sexp.cpp

```diff
@@ -819,6 +819,6 @@ bool multi_get_ushort(ushort &value)
 void multi_discard_remaining_callback_data()
 {
 	if (!multi_sexp_discard_operator()) {
-		Warning(LOCATION, "Attempt to discard remaining data failed! Callback for operator %d lacks proper termination. Entire packet may be corrupt. Discarding remaining packet"); 
+		Warning(LOCATION, "Attempt to discard remaining data failed! Callback for operator lacks proper termination. Entire packet may be corrupt. Discarding remaining packet");
```

Comment:
FRED should check if the jump node actually exists in the mission or not when saving a mission so I initially thought that the lookup was unlikely to fail. But then I noticed that there is a SEXP for changing the name of a jump node. That actually makes it pretty easy to make the lookup fail on the server. Name changes should propagate over to the clients too, but if somehow one doesn't, you've basically got a fairly large error if you don't clean up the packet and simply ignore it. 

The multi_discard_remaining_callback_data() function is only used by a couple of functions at the moment, but it probably could be used elsewhere if you have errors in the data the client has received. I think displaying the op_number is enough data to make a start on fixing bugs. Displaying the op name would probably be better though.